src/framework/core/js/DataBinding.js

```diff
@@ -329,11 +385,47 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
         });
     };
 
+    /** Compute relay dependency out arcs for a group of initialising components.
+     * @param transacs {Object} Hash of component id to local ChangeApplier transaction
+     * @param mrec {Object} Hash of component id to enlisted component record
+     * @return Object Hash of component id to list of enlisted component record
+     */
+    fluid.computeInitialOutArcs = function (transacs, mrec) {
+        return fluid.transform(mrec, function (recel, id) {
+            var oneOutArcs = {};
+            var listeners = recel.that.applier.listeners.sortedListeners;
+            fluid.each(listeners, function (listener) {
+                console.log("oIT got listener ", listener);
+                if (listener.isRelay && !fluid.isExcludedChangeSource(transacs[id], listener.cond)) {
+                    var targetId = listener.targetId;
+                    if (targetId !== id) {
+                        oneOutArcs[targetId] = true;
+                    }
+                    console.log("Unexcluded arc to ", targetId, " with record ", mrec[targetId]);
```

Comment:
As above—remove logging if possible.

Reply:
Sorry to miss these stray logs, now removed